src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -627,6 +627,16 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
   private pasteDataValidation(origin: CellPosition, target: CellPosition) {
     const rule = this.getters.getValidationRuleForCell(origin);
     if (!rule) {
+      const targetRule = this.getters.getValidationRuleForCell(target);
```

Comment:
Is this related to the new feature? I feel like it's more of a bug fix or maybe the situation could not occur beforehand?

Reply:
It's not really a bug fix but a change in the desired behavior we discussed with Francois. But it should probably be added in a separated commit